src/components/App/index.js

```diff
@@ -1,42 +1,55 @@
 import React, { Component } from "react";
 import { drawContributions } from "github-contributions-canvas";
-import { download } from "./utils/export";
-import loadingImage from "./loading.gif";
-import "./App.css";
+import { download } from "../../utils/export";
+import GithubContribution from "../../services/github-contribution";
+import InputField from "../InputField";
+import Button from "../Button";
+import GithubStarLink from "../GithubStarLink";
+import ErrorResponse from "../ErrorResponse";
+import Loading from "../Loading";
+import ContributionGraph from "../ContributionGraph";
 
 class App extends Component {
-  canvas = null;
-  availableThemes = {
-    standard: "GitHub",
-    halloween: "Halloween",
-    teal: "Teal",
-    leftPad: "@left_pad",
-    dracula: "Dracula",
-    blue: "Blue",
-    panda: "Panda 🐼"
-  };
+  constructor (props) {
+    super();
 
-  state = {
-    loading: false,
-    data: null,
-    error: null,
-    username: "",
-    theme: "standard"
-  };
+    this.canvas = null;
```

Comment:
same, move this outside of constructor as it was previously

Reply:
@catalinmiron Moved to outside of constructor as it was previously :)